client/components/Layout/Layout.tsx

```diff
@@ -18,7 +18,7 @@ require('./layout.scss');
 type Props = LayoutOptions & {
 	blocks: LayoutBlock[];
 	id?: string;
-	pubs: Pub[];
+	pubs: DefinitelyHas<Pub, 'releases'>[];
```

Comment:
What are your thoughts on creating a type like `PubWithReleases` that we can import instead of having to write this type each time?

Reply:
I don't know yet. I'd actually almost prefer to make `releases` non-nullable on `Pub` since it seems like the kind of thing you need practically every time you query it — thoughts?

Reply:
Yep, I agree it should probably be non-nullable/eagerly loaded as long as the Release objects aren't too hefty, which they don't appear to be.